model/src/main/java/com/cloudera/nav/sdk/model/relations/RelationValidator.java

```diff
@@ -61,4 +61,15 @@ public void validateRequiredMProperties(Relation relation) {
       }
     }
   }
+
+  public void validatateRelation(Relation relation) {
+    Preconditions.checkArgument(
```

Comment:
Are there other relations validation criteria other than these? If you want to add additional single field validation criteria you'd have to explicitly check for it now?